docs/sample_config/default_config.nu

```diff
@@ -20,10 +20,10 @@ let-env PROMPT_COMMAND_RIGHT = { create_right_prompt }
 
 # The prompt indicators are environmental variables that represent
 # the state of the prompt
-let-env PROMPT_INDICATOR = "〉"
-let-env PROMPT_INDICATOR_VI_INSERT = ": "
-let-env PROMPT_INDICATOR_VI_NORMAL = "〉"
-let-env PROMPT_MULTILINE_INDICATOR = "::: "
+let-env PROMPT_INDICATOR = { build-string 〉}
```

Comment:
Wouldn't just `{ '〉' }` be enough without the `build-string`?

Reply:
Mmm... Probably